stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -94,13 +108,13 @@ public fun MessageText(
         Text(
             modifier = modifier
                 .padding(
-                    start = 12.dp,
-                    end = 12.dp,
+                    start = if (isEmojiOnly && !isMyMessage) 0.dp else 12.dp,
+                    end = if (isEmojiOnly && isMyMessage) 0.dp else 12.dp,
                     top = 8.dp,
                     bottom = 8.dp
```

Comment:
I suggest having the following check here:

```
top = if (isEmojiOnly) 0.dp else 8.dp,
bottom = if (isEmojiOnly) 0.dp else 8.dp,
```

As currently the font padding adds too much of empty space around the message by default and if we add `8.dp` padding on top of it, it just looks weird

Reply:
![padding](https://user-images.githubusercontent.com/17215808/164381562-e8c532ea-4948-4564-be23-97c93b007fd3.png)